src/openag_brain/load_env_var_types.py

```diff
@@ -0,0 +1,37 @@
+class VariableInfo(dict):
```

Comment:
Originally I set it up to use a dictionary. However, there are numerous places that it is referenced directly or as a key value lookup. For example,             
video_writer.py: 
    self.environment, "measured", variable.name,

I started going through and changing them all to use a dictionary, but later thought it would be more seamless of a change just to mimic the class structure so it could be used in the same way as it was previously.

thoughts?